core/src/main/java/org/apache/gravitino/hook/AccessControlHookDispatcher.java

```diff
@@ -192,9 +191,10 @@ public boolean deleteRole(String metalake, String role) throws NoSuchMetalakeExc
       log.debug(e.getMessage());
     }
     boolean resultOfDeleteRole = dispatcher.deleteRole(metalake, role);
-    if (oldRole != null) {
-      notifyRoleUserRelChange(((RoleEntity) oldRole).id());
-    }
+
+    if(resultOfDeleteRole && oldRole!= null) 
+      notifyRoleUserRelChange(metalake,role)
+
```

Comment:
The CI is failing here because we use Spotless to make sure the code is consistently formatted. You can run Spotless yourself to fix this or manually add the brackets back in.

Reply:
I'm also not sure why you have changed the parameters